tests/run_tests.py

```diff
@@ -84,6 +84,11 @@
   "opencl_read_and_write_image3d_rgba32i.amber",
   "opencl_write_image.amber",
   "glsl_read_and_write_image3d_rgba32i.amber",
+   "compute_require_full_subgroups.amber",
```

Comment:
Can you add a comment for why these are disabled?

Reply:
I removed changes in this file.